src/main/kotlin/com/openlattice/search/SortDefinition.kt

```diff
@@ -0,0 +1,37 @@
+package com.openlattice.search
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.google.common.base.Preconditions
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
+
+data class SortDefinition(
+        @JsonProperty(SerializationConstants.TYPE_FIELD) val sortType: SortType = SortType.score,
+        @JsonProperty(SerializationConstants.IS_DESCENDING) val isDescending: Boolean = true,
+        @JsonProperty(SerializationConstants.PROPERTY_TYPE_ID) val propertyTypeId: Optional<UUID> = Optional.empty(),
+        @JsonProperty(SerializationConstants.LATITUDE) val latitude: Optional<Double> = Optional.empty(),
+        @JsonProperty(SerializationConstants.LONGITUDE) val longitude: Optional<Double> = Optional.empty()
+) {
+
+    init {
+
+        when (sortType) {
+
+            SortType.field -> Preconditions.checkNotNull(
```

Comment:
Do you not have yellow here? I'm pretty sure that this can never be null.

Reply:
I did not :( but yeah I don't think that could've been null